src/lib/components/icon/CustomIcon.tsx

```diff
@@ -0,0 +1,1261 @@
+        fill="currentColor"
+      />
+    ),
+    viewBox: viewboxDefault,
+  },
+  "wallet-solid": {
+    svg: (
+      <path
+        fillRule="evenodd"
+        clipRule="evenodd"
+        d="M10.6771 0.692542C10.2129 0.0159609 9.29071 -0.1988 8.5614 0.199855L3.07295 3.19995H1.6C0.716344 3.19995 0 3.9163 0 4.79995V14.4C0 15.2836 0.716345 16 1.6 16H14.4C15.2837 16 16 15.2836 16 14.4V12.8H8.8C7.91634 12.8 7.2 12.0836 7.2 11.2V7.99998C7.2 7.11632 7.91634 6.39998 8.8 6.39998H16V4.79995C16 3.9163 15.2837 3.19995 14.4 3.19995H12.3973L10.6771 0.692542ZM10.4743 3.19995L9.34582 1.55501L6.33652 3.19995H10.4743ZM16 7.99998H9.8C9.24771 7.99998 8.8 8.44769 8.8 8.99998V10.2C8.8 10.7523 9.24772 11.2 9.8 11.2H16V7.99998ZM11.2 9.59998C11.2 10.0418 10.8418 10.4 10.4 10.4C9.95817 10.4 9.6 10.0418 9.6 9.59998C9.6 9.15815 9.95817 8.79998 10.4 8.79998C10.8418 8.79998 11.2 9.15815 11.2 9.59998Z"
+        fill="currentColor"
+      />
+    ),
+    viewBox: viewboxDefault,
+  },
+  website: {
+    svg: (
+      <path
+        d="M7.992 0C3.576 0 0 3.584 0 8C0 12.416 3.576 16 7.992 16C12.416 16 16 12.416 16 8C16 3.584 12.416 0 7.992 0ZM13.536 4.8H11.176C10.92 3.8 10.552 2.84 10.072 1.952C11.544 2.456 12.768 3.48 13.536 4.8ZM8 1.632C8.664 2.592 9.184 3.656 9.528 4.8H6.472C6.816 3.656 7.336 2.592 8 1.632ZM1.808 9.6C1.68 9.088 1.6 8.552 1.6 8C1.6 7.448 1.68 6.912 1.808 6.4H4.512C4.448 6.928 4.4 7.456 4.4 8C4.4 8.544 4.448 9.072 4.512 9.6H1.808ZM2.464 11.2H4.824C5.08 12.2 5.448 13.16 5.928 14.048C4.456 13.544 3.232 12.528 2.464 11.2V11.2ZM4.824 4.8H2.464C3.232 3.472 4.456 2.456 5.928 1.952C5.448 2.84 5.08 3.8 4.824 4.8V4.8ZM8 14.368C7.336 13.408 6.816 12.344 6.472 11.2H9.528C9.184 12.344 8.664 13.408 8 14.368ZM9.872 9.6H6.128C6.056 9.072 6 8.544 6 8C6 7.456 6.056 6.92 6.128 6.4H9.872C9.944 6.92 10 7.456 10 8C10 8.544 9.944 9.072 9.872 9.6ZM10.072 14.048C10.552 13.16 10.92 12.2 11.176 11.2H13.536C12.768 12.52 11.544 13.544 10.072 14.048V14.048ZM11.488 9.6C11.552 9.072 11.6 8.544 11.6 8C11.6 7.456 11.552 6.928 11.488 6.4H14.192C14.32 6.912 14.4 7.448 14.4 8C14.4 8.552 14.32 9.088 14.192 9.6H11.488Z"
+        fill="currentColor"
+      />
+    ),
+    viewBox: viewboxDefault,
+  },
+};
+
+interface CustomIconProps extends IconProps {
+  color?: string;
```

Comment:
'color' and 'boxSize' are available in 'IconProps'. So we can remove them from 'CustomIconProps'.